### PR TITLE
KF 1.9 release: fix date and link to video

### PR DESCRIPTION
**Pull request by tarilabs**
1. fix the release date,
ref: https://github.com/kubeflow/manifests/releases/tag/v1.9.0
as noticed by @thesuperzapper 

2. link to Kubeflow 1.9 Release Update youtube video

👉 if the PR is approved, I will update the video from Unlisted -> Published

@kubeflow/release-team @rimolive @StefanoFioravanzo @andreyvelich @hbelmiro @jbottum @juliusvonkohout

### CODE REVIEW

**Comment by rimolive** [comment]
Thank you Matteo for working in the release video!

/lgtm

**Review by hbelmiro — COMMENTED** [review]
Thank you @tarilabs.
/lgtm

**Comment by thesuperzapper** [comment]
@tarilabs In the interest of fixing the incorrect date while we wait for the video to be ready(we need to upload the video to the `Kubeflow` channel before we publish, not the `Kubeflow Community`), I have raised a separate PR to only fix the date:

- https://github.com/kubeflow/website/pull/3840

File: content/en/docs/releases/kubeflow-1.9.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ weight = 95
           <a href="https://blog.kubeflow.org/kubeflow-1.9-release/">Kubeflow 1.9 Release Announcement</a>
         <br>
         <b>Video:</b> 
-          To be published
+          <a href="https://www.youtube.com/watch?v=3sZzE-vPsMI">Kubeflow 1.9 Release Update</a>
```
Comment (andreyvelich) [inline content/en/docs/releases/kubeflow-1.9.md:25]: Please can you upload this video to the Kubeflow YouTube channel: https://www.youtube.com/kubeflow
@james-jwu and @zijianjoy should help you with that.

Comment (tarilabs) [reply content/en/docs/releases/kubeflow-1.9.md]: done with 207197a

**Comment by tarilabs** [comment]
can this be approved, please? the video link is indeed referring to the video on the main Kubeflow channel.

unrelated, this:
- https://github.com/kubeflow/community/issues/745

**Pull request merged**